src/calculateRentalCost.js

```diff
@@ -28,7 +28,7 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  return days >= 7 ? days * 40 - 50 : days >= 3 ? days * 40 - 20 : days * 40;
```

Comment:
ternary operator is not the best idea for this
Also one ternary inside another ternary makes code unreadable

It is better to use if conditions here